app/src/test/java/edu/ucandroid/weathernotice/CountryUnitTest.kt

```diff
@@ -44,7 +38,7 @@ class ExampleUnitTest {
     fun weatherDTO_containsCincinnati() {
         givenViewModelIsInitialized()
         whenJSONDataAreReadAndParsed()
-        thenResultsShouldContainBelize()
+        thenResultsShouldContainCincinnati()
```

Comment:
Refactored to better reflect what is being tested here.

Reply:
I think we may end renaming that but will take the change.